src/data/custom-fonts.ts

```diff
@@ -0,0 +1,7 @@
+export const feedbackCustomFonts: { [language: string]: string } = {
+  // english: "Consolas", // sample. if in future we need to customize feedback text in specific language
+};
+
+export const titleTextCustomFonts: { [language: string]: string } = {
```

Comment:
These are good idea but why can we use this right now? Why can't we put the defaults we are using?

It's just doesn't make sense that you made a function to change the fonts but how are we using the original font? why can't we just call it here and have all the font changing be done here?

Reply:
I used this file and set a font in every languages. pls check @bernhardccodev ebb9dad35ef40ffc1d67981b4b037c79a2f36384

Reply:
You just made a commit to use this, right?

Also please use your codev and refrain from doing commits with your personal email
![Uploading image.png…]()

Reply:
yes. thanks for the heads up. @bernhardccodev

Reply:
@woodsjohn-codev how does titleTextCustomFonts and  feedbackCustomFonts works? Can we remove the default font declared in the constants `export const defaultFont = "Atma-SemiBold"`; if we have this two functions for fonts?

Reply:
@bernhardccodev the purpose of titleTextCustomFonts and feedbackCustomFonts is to set a custom font incase the defaultFont dont support the other language if we add a new language in the future. so if ever theres a config inside of titleTextCustomFonts and feedbackCustomFonts it will use the font assigned. if not it will just use the defaultFont which is Atma

Reply:
But can we remove the default fonts declared in constants and just use the default one through this function?

Reply:
`private async loadFont() {
    const fontOptions = {
      customFonts: feedbackCustomFonts,
defaultFont: feedbackCustomFonts.default //instead of default
      lang: lang,
    };
    applyFontToElement(this.feedbackTextElement, fontOptions, FONT_BASE_PATH);
  } `

Reply:
@bernhardccodev done i applied it already. thanks